docs/items/loot_tables.md

```diff
@@ -0,0 +1,170 @@
+                    "type": "item",
+                    "name": "minecraft:nether_star",
+                    "weight": 40
+                },
+                {
+                    "type": "empty",
+                    "weight": 60
+                }
+            ]
+        }
+    ]
+}
+```
+
+!!! Note
+    You still need to register this table with `LootTableList.register()`!
+
+Then the loot entry and pool are created and added, resulting in a new loot pool for dungeon chests that has a 60% chance of nothing and 40% of a nether star.
+```Java
+LootEntry entry = new LootEntryTable(new ResourceLocation("mymod:inject/simple_dungeon"), <weight>, <quality>, <conditions>, <entryName>); // weight doesn't matter since it's the only entry in the pool. Other params set as you wish.
+
+LootPool pool = new LootPool(new LootEntry[] {entry}, <conditions>, <rolls>, <bonusRolls>, <name>); // Other params set as you wish.
+
+evt.getTable().addPool(pool);
+```
+
+Of course, if the loot you want to add cannot be determined ahead of time, you can freely construct and add `LootPool`s and implementations of `LootEntry` in your event handler similar to the calls shown above.
+
+A real-world example of this approach in action can be seen in Botania. The event handler is located [here](https://github.com/Vazkii/Botania/blob/e38556d265fcf43273c99ea1299a35400bf0c405/src/main/java/vazkii/botania/common/core/loot/LootHandler.java), and the injected tables are located [here](https://github.com/Vazkii/Botania/tree/e38556d265fcf43273c99ea1299a35400bf0c405/src/main/resources/assets/botania/loot_tables/inject).
+
+Changing mob drops
+------------------
+
+Subclasses of `EntityLiving` automatically support drawing from a loot table upon death. This is done by overriding the `getLootTable` method to return a `ResourceLocation` to the desired table. This serves as the mob's default table; the tables of both your and other mods' mobs can be overridden for a single entity by setting the `deathLootTable` field of the entity. 
+
+Generating loot in-code
+-----------------------
+
+Occasionally, you may want to generate `ItemStack`s from a loot table from your own code.
+
+First, obtain the loot table itself (you need access to a `World`):
+```Java
+LootTable table = this.world.getLootTableManager().getLootTableFromLocation(new ResourceLocation("mymod:my_table")); // resolves to /assets/mymod/loot_tables/my_table.json
+```
+
+Next, create a `LootContext` using the provided `LootContextBuilder`, which holds information about the context of the looting, such as the killer, luck of the looter, and finishing blow.
+```Java
+LootContext ctx = new LootContext.Builder(world)
+    .withLuck(...) // adjust luck, commonly EntityPlayer.getLuck()
+    .withLootedEntity(...) // set looted entity
+    .withPlayer(...) // set player as killer
+    .withDamageSource(...) // pass killing blow and non-player killer
+    .build();
+```
+
+Finally, to get a collection of `ItemStack`s:
```

Comment:
This is kind of an excessive code block. I think this would be just as good as doing

"Finally, to get a `List` of `ItemStack`s, call `LootTable#generateLootForPools(Random, Context??)`."